go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/Microsoft/go-winio v0.6.1
 	github.com/Microsoft/hcsshim v0.11.4
-	github.com/awslabs/soci-snapshotter v0.4.1
+	github.com/awslabs/soci-snapshotter v0.0.0-20240123154419-8d2b306690f9
```

Comment:
Thanks, can we have a new tag?

Reply:
@AkihiroSuda , sure I'll help get the ball started on that on the SOCI side. Can we close this PR and #2761 so dependabot can catch v0.5.1 when it is cut?